Documentation/ApiOverview/Environment/ApplicationContext.rst

```diff
@@ -0,0 +1,383 @@
+:orphan:
+
+..  include:: /Includes.rst.txt
+..  index:: ! Application Context
+
+..  _application-context:
+
+===================
+Application Context
+===================
+
```

Comment:
Please add a TOC: this page is very comprehensive and one has to scroll down to get what's all in there. A TOC helps - and also provides anchors to jump directly to the needed section.

Reply:
I don't know how to do that. `toctree` reads the header of multiple files. This here is just one file. Using toctree would, in that case, add just one entry with the headline of this one file.

Reply:
He is talking about the `..  contents::` directive: https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/Reference/ReStructuredText/Menus/ContentMenu.html

Reply:
I added the table of contents.